lib/middleman-images/manipulator.rb

```diff
@@ -65,7 +65,7 @@ def ignore_orginal_resources(resources)
           if unused_originals.include? resource.source_file
             resource.ignore!
           elsif required_originals.include? resource.source_file
-            resource.ignored = false
+            resource.instance_variable_set(:@ignored, false)
```

Comment:
It's a shame that they removed the accessor.

Reply:
They obviously don't want us to unignore ressources. So at some point we might need to change this into something where we do not need to do so. But for now this seems to work.